concepts/05 UI Components/SelectBox/00 Getting Started with SelectBox/00 Getting Started with SelectBox.md

```diff
@@ -1,5 +1,3 @@
-#include tutorials-intro-installationnote
-
 SelectBox is an editor that allows users to select a value from a drop-down list or add a new value.
 
 In this tutorial, we will create the SelectBox and configure its basic features. The created UI component has a populated drop-down list, allows users to search through it, and logs the previous and current selected items to the console.
```

Comment:
>Does it mean that the created component has a search box? If yes, it would be much better if you wrote something like this:

The search works as follows: users type some text in the select/tagbox and then it filters items based on the input:

![image](https://user-images.githubusercontent.com/33040934/124272558-91c99400-db47-11eb-8e6a-abe7f34e71ec.png)
![image](https://user-images.githubusercontent.com/33040934/124272580-98f0a200-db47-11eb-925c-c716d3cf2954.png)

Reply:
Alright, if you've already used this sentence before and it worked, I see no strong reason to change it.